examples/run/run.cpp

```diff
@@ -34,14 +36,17 @@
 }
 #endif
 
+#define LLAMA_USE_CURL
```

Comment:
This should be removed before we merge, compiler options are supposed to decide if this is defined or not.

Reply:
Ups, missed to remove that one. Removed it now.